### PR TITLE
[GooglePlayReleaseBundle] Add input parameter to define Release-Name

### PR DESCRIPTION
**Pull request by tatyana-kostromskaya**
**Task name**: GooglePlayReleaseBundle

**Description**: Add input "Release Name" for Bundle Release task. 

**Documentation changes required:** (Y/N) Y

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) #257 

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/google-play-vsts-extension/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by DaniilShmelev — COMMENTED** [review]
Should the tests also be modified?

File: Tasks/GooglePlayReleaseBundle/task.loc.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   ],
   "version": {
     "Major": "3",
-    "Minor": "189",
+    "Minor": "191",
```
Comment (anatolybolshakov) [inline Tasks/GooglePlayReleaseBundle/task.loc.json:17]: Please bump to 192

**Comment by tatyana-kostromskaya** [comment]
> Should the tests also be modified?

@DaniilShmelev , I am not sure that I need to change tests, because I didn't change the main logic and this input affects only view of the released bundle in the Google Console. Additionally, this input could not break execution of the task.
So I don't know what exactly new tests may validate

**Review by EzzhevNikita — APPROVED** [review]

**Review by DaniilShmelev — APPROVED** [review]

**Review by anatolybolshakov — APPROVED** [review]
LGTM, thanks!

**Review by DaniilShmelev — APPROVED** [review]

**Pull request merged**